src/script/conversation/ConversationRepository.ts

```diff
@@ -2806,7 +2806,11 @@ export class ConversationRepository {
 
     const inSelfConversation = this.conversationState.isSelfConversation(conversationId);
     if (inSelfConversation) {
-      const typesInSelfConversation = [CONVERSATION_EVENT.MEMBER_UPDATE, ClientEvent.CONVERSATION.MESSAGE_HIDDEN];
+      const typesInSelfConversation = [
+        CONVERSATION_EVENT.MEMBER_UPDATE,
+        ClientEvent.CONVERSATION.MESSAGE_HIDDEN,
+        CLIENT_CONVERSATION_EVENT.E2EI_VERIFICATION,
+      ];
```

Comment:
Question: would it be possible to not inject this event in the selfConversation (upstream)? 
It doesn't really make sense to send this event in the self conv

Reply:
Good idea, done with https://github.com/wireapp/wire-webapp/pull/16554/commits/b7fb6026f4e42bdad19e8f0290be5193903825de